### ISSUE TITLE
Verify error:"error":{"type":"urn:acme:error:unauthorized","detail":"

### ISSUE THREAD

**Issue by c3x** [Neilpang/acme.sh#1569]
Hello!
I try to get an certificate for one of my domains.
This domain was blocked by Google in case of Malware. I had register the domain a few days before so I don't know about the problems.
I send an report to Google and the domain is already unblocked.

But I still get the error, that the domain is blocked by third party api.

How to fix this issue?
Is it a problem by ACME or LetsEncrypt?

**Comment by neilpang** [comment]
Closing as stale: no activity for over 18 months. If this still applies on the current release, please open a new issue with the `--debug 2` log.

**Comment by neilpang** [comment]
Not a bug: Let's Encrypt refused validation because the domain was flagged by Google Safe Browsing; that is CA-side policy, not acme.sh.
